app/admin/sponsor.rb

```diff
@@ -32,7 +32,7 @@
     column :request_fulfilled
     column :sponsor_type
     column :country do |_sponsor|
-      "(#{ISO3166::Country[_sponsor.country]}) #{t _sponsor.country, locale: :ar}"
+      "(#{ISO3166::Country[_sponsor.country]}) #{t _sponsor.country, locale: :ar, default: ''}"
```

Comment:
That's why you should keep reading the docs even after you've found the answer :) :+1: